### DISCUSSION TITLE
I have a react-hook-form, I have to submit the form by clicking the tick icon in header.how can I achieve it. In Controller component

### DISCUSSION THREAD

**Discussion by mohanagowthami** [react-hook-form/react-hook-form#2696] [Q&A]
I have a react-hook-form, I have to submit the form by clicking the tick icon in header.how can I achieve it. In Controller component

**Comment by bluebill1049** [comment]
handleSubmit()()

**Reply by mohanagowthami** [reply]
i did not have access to the **handleSubmit**  method

**Reply by bluebill1049** [reply]
make it, so you can access it...
